build/gulpfile.vscode.js

```diff
@@ -528,7 +528,8 @@ gulp.task('vscode-translations-push-test', ['optimize-vscode'], function () {
 		gulp.src(pathToMetadata).pipe(i18n.createXlfFilesForCoreBundle()),
 		gulp.src(pathToSetup).pipe(i18n.createXlfFilesForIsl()),
 		gulp.src(pathToExtensions).pipe(i18n.createXlfFilesForExtensions())
-	).pipe(i18n.findObsoleteResources(apiHostname, apiName, apiToken)
+	// {{SQL CARBON EDIT}}
```

Comment:
might be worth adding a comment here about why this is being disabled

Reply:
Done.  The function is being disable since it calls the Transifex API using VS Code context.  We're not using Transifex, so we output the XLF files to local disk only.